Author: <thijs@debian.org>
Description: No description.
Index: b/judge/judgedaemon.main.php
===================================================================
--- a/judge/judgedaemon.main.php	2012-05-20 23:42:10.125788008 +0200
+++ b/judge/judgedaemon.main.php	2012-05-21 09:50:02.061357824 +0200
@@ -44,7 +44,8 @@
 if ( isset($options['V']) ) version();
 if ( isset($options['h']) ) usage();
 
-setup_database_connection('jury');
+#setup_database_connection('jury');
+require(LIBJUDGEDIR . '/debian-database.php');
 
 $verbose = LOG_INFO;
 if ( isset($options['verbose']) ) $verbose = $options['verbose'];
