Author: <thijs@debian.org>
Description: No description.
--- a/lib/use_db.php
+++ b/lib/use_db.php
@@ -4,6 +4,7 @@
 
 function setup_database_connection()
 {
+/*
 	$credfile = ETCDIR . '/dbpasswords.secret';
 	$credentials = @file($credfile);
 	if (!$credentials) {
@@ -37,5 +38,8 @@
 		user_error("Failed to create database connection", E_USER_ERROR);
 		exit();
 	}
+*/
+	$credfile = ETCDIR . '/domserver.dbconfig.php';
+	require(LIBDIR . '/debian-database.php');
 }
 
