Author: <thijs@debian.org>
Description: No description.
--- a/judge/judgedaemon.main.php
+++ b/judge/judgedaemon.main.php
@@ -54,7 +54,9 @@
 define ('LOGFILE', LOGDIR.'/judge.'.$myhost.'.log');
 require(LIBDIR . '/init.php');
 
-setup_database_connection();
+#setup_database_connection();
+$credfile = ETCDIR . '/judgedaemon.dbconfig.php';
+require(LIBDIR . '/debian-database.php');
 
 $verbose = LOG_INFO;
 if ( isset($options['verbose']) ) {
--- a/lib/use_db.php
+++ b/lib/use_db.php
@@ -4,6 +4,7 @@
 
 function setup_database_connection()
 {
+/*
 	$credfile = ETCDIR . '/dbpasswords.secret';
 	$credentials = @file($credfile);
 	if (!$credentials) {
@@ -37,5 +38,8 @@
 		user_error("Failed to create database connection", E_USER_ERROR);
 		exit();
 	}
+*/
+	$credfile = ETCDIR . '/domserver.dbconfig.php';
+	require(LIBDIR . '/debian-database.php');
 }
 
